doc/users/dflt_style_changes.rst

```diff
@@ -18,6 +18,12 @@ values is a single line of python
 See :ref:`customizing-with-matplotlibrc-files` for details about how to
 persistently and selectively revert many of these changes.
 
+.. note::
```

Comment:
I would give this a `'YTick alignment'` section in the `'Tick Label Formatting'` section.   We don't have completely throw our selves under the bus in the first section.  I don't think this needs to be called out in a note, merging it into the paragraph above would be enough.